README.md

```diff
@@ -51,6 +51,15 @@ build: 'Build-Related',
 breaking: 'Breaking Changes'
 ```
 
+However, you can also create a user-specified mapping by creating a file `pr-log.json` in your project's root directory containing an alternative mapping. For example:
```

Comment:
I don’t have a strong opinion about this, but just asking if `pr-log.json` would be the best place? Maybe we could use the `package.json` to look up for this information?